Makefile

```diff
@@ -41,7 +41,7 @@ vendor:
 install-pact:
 	@if [ ! -d ./pact ]; then \
         echo "pact not installed, installing..."; \
-        wget --quiet https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v${PACT_VERSION}/${PACT_FILE} -O /tmp/pactserver.tar.gz && tar -xzf /tmp/pactserver.tar.gz 2>/dev/null -C .; \
+        curl -s https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v${PACT_VERSION}/${PACT_FILE} -L -o /tmp/pactserver.tar.gz && tar -xzf /tmp/pactserver.tar.gz 2>/dev/null -C .; \
```

Comment:
While we're changing this, can we do `-sSLf`

```
       -s, --silent
      -S, --show-error
              When used with -s, --silent, it makes curl show an error message if it fails.
       -L, --location
       -f, --fail
```

perhaps all the options directly after curl, rather it being in the middle?  `-o` could remain where it is I guess.

Reply:
Done :+1: